eng/pipelines/mgmt-auto-release.yml

```diff
@@ -0,0 +1,54 @@
+trigger: none
+
+pr: none
+
+schedules:
+  - cron: "0 1 * * *"
+    displayName: Daily release
+    branches:
+      include:
+        - main
+    always: true
+
+stages:
+  - stage:
+    jobs:
+      - job:
+        displayName: Go Auto Release
+        steps:
+          - task: Npm@1
+            inputs:
+              command: 'custom'
+              customCommand: 'install -g autorest@latest'
+
+          - task: GoTool@0
+            inputs:
+              version: '1.18.2'
+
+          - script: |
+              export PATH=$PATH:$HOME/go/bin
+              sudo ln -s /usr/bin/pwsh /usr/bin/pwsh.exe
+              git config --global user.email "ReleaseHelper"
+              git config --global user.name "ReleaseHelper"              
+              
+              cd ../
+              git clone https://github.com/Azure/azure-sdk-for-go.git
+              git clone https://github.com/Azure/azure-rest-api-specs.git
+              
+              cd azure-sdk-for-go
+              git remote add fork https://Azure:$(azuresdk-github-pat)@github.com/Azure/azure-sdk-for-go.git
```

Comment:
Why `fork` out of curiosity?

Reply:
In this [PR](https://github.com/Azure/azure-sdk-for-go/pull/18455) we changed the generator tools and added 'git remote fork' to facilitate submitting PRs from the personal fork repository to the origin repository